exampleAlgorithm.tex

```diff
@@ -28,8 +28,8 @@ \chapter{Example Algorithm} \label{Algorithm}
      the subsequent synchronization packet, which does not report branch map history;
    \item \textit{er\_ccdn.}  Instruction retirement and exception signalled on the same cycle, 
      or context has changed and should be treated as an uninferable PC discontinuity, or
-     context notify (see Table~\ref{tab:context-type});
-   \item \textit{exc\_only.}  Exception signaled without simultaneous retirement;
+     context notification (see Table~\ref{tab:context-type});
+   \item \textit{exc\_only.}  Exception signalled without simultaneous retirement;

```

Comment:
There are two correct spellings of signaled/signalled but this makes it consistent with all other uses in the document.